src/plugins/laser-cluster/laser-cluster-thread.cpp

```diff
@@ -452,13 +452,14 @@ LaserClusterThread::loop()
 
     //logger->log_info(name(), "Found %zu clusters", cluster_indices.size());
 
-    unsigned int i = 0;
+    //unsigned int i = 0;
```

Comment:
ditto

Reply:
If the debug output a few lines later is uncommented this needs to be added back in, too.